master/internal/resourcemanagers/agent_resource_manager.go

```diff
@@ -294,6 +294,7 @@ func (a *agentResourceManager) createResourcePoolSummary(
 	instanceType := ""
 	slotsPerAgent := -1
 	slotType := device.ZeroSlot
+	accelerator := "-"
```

Comment:
If I am not mistaken, `gceInstanceType.GPUType` will return an empty string when there's not accelerator. I'd suggest using empty string to denote an "empty" value here and in `AWS.Accelerator()` method. If webui needs to display "-" or "N/A" or other special visual treatment, then it can be done on client side.